src/components/map/MapFilter.js

```diff
@@ -11,7 +11,7 @@ import { StyledDiv, StyledFilterDiv } from '../../styles/StyledComponents';
 const MapFilter = () => {
   useEffect(() => {
     axios
-      .get(process.env.REACT_APP_BACKENDURL)
+      .get(`${process.env.REACT_APP_BACKENDURL}/incidents/showallincidents`)
```

Comment:
this change was mad because the previous `REACT_APP_BACKENDURL ` included `/incidents/showallincidents` which was a problem